### ISSUE TITLE
ASSWipe - LineBoundsBatch error, not sure of cause

### ISSUE THREAD

**Issue by Fyurie** [TypesettingTools/line0-Aegisub-Scripts#6]
```Moonscript
Lua reported a runtime error:
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\autoload\l0.ASSWipe.moon", line 289
<anonymous function at lines 131-327>
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\include\a-mo\LineCollection.moon", line 182
runCallback
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\autoload\l0.ASSWipe.moon", line 225
callback
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\include\l0\ASSFoundation\LineContents.lua", line 136
callback
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\autoload\l0.ASSWipe.moon", line 219
callback
    File "C:/Users\Fyurie\AppData\Roaming\Aegisub\automation\autoload\l0.ASSWipe.moon", line 182
removeInvisibleContoursOpt
attempt to call field 'LineBoundsBatch' (a nil value)
```

Lines processed to cause this error were:
```
Dialogue: 0,0:04:42.17,0:04:44.18,FyuSigns,,0,0,0,,{\bord1.5\blur0.6\shad1\an7\pos(0,0)\fscx100\fscy100\p1\3c&H2E2E2F&\c&H2E2E2F&\4c&H000000&\4a&HA0&}m 394 168 l 395 197 656 150 656 119
Dialogue: 1,0:04:42.17,0:04:44.18,FyuSigns,,0,0,0,,{\bord0\blur1\an7\pos(0,0)\fscx100\fscy100\p1\c&HB5B5B5&\4c&H000000&\4a&HA0&\shad1\4a&HFF&}m 394 168 l 395 197 656 150 656 119
Dialogue: 1,0:04:42.17,0:04:44.18,FyuSigns,,0,0,0,,{\c&H3C3C3C&\fs18\blur0.6\fnCorporate Rounded\b1\fax-0.18\pos(526.451,157.775)\frz10.37}T{\fax-0.18}o{\fax-0.18}k{\fax-0.18}y{\fax-0.18}o {\fax-0.18}C{\fax-0.18}i{\fax-0.18}t{\fax-0.18}y {\fax-0.18}D{\fax-0.18}i{\fax-0.18}s{\fax-0.18}a{\fax-0.18}s{\fax-0.18}t{\fax-0.18}e{\fax-0.18}r {\fax-0.18}P{\fax-0.18}r{\fax-0.18}e{\fax-0.18}v{\fax-0.18}e{\fax-0.18}n{\fax-0.18}t{\fax-0.18}i{\fax-0.18}o{\fax-0.18}n {\fax-0.18}S{\fax-0.18}t{\fax-0.18}o{\fax-0.18}r{\fax-0.18}a{\fax-0.18}g{\fax-0.18}e
```

**Comment by line0** [comment]
This is caused by a recent ASSFoundation update overwriting the experimental version you were using before as a part of an experimental ASSWipe package i've sent you for testing quite some time ago.

I'm working on releases for all involved components, which need to be deployed at the same time to be compatible. In the meantime you could delete your local copy off ASSWipe and install the official version from DepCtrl. This will lose you some features of the experimental version (performance, extradata cleaning and quirks mode) but at least it works.

**Comment by Fyurie** [comment]
Yup, all good on my end for now. Looking forward to the update restoring my old functionality.

**Comment by line0** [comment]
Fixed w/ ASSWipe v0.4.0 and ASSFoundation v0.4.0.
